app/models/page.rb

```diff
@@ -10,9 +10,24 @@ class Page < ApplicationRecord
 
   # FIXME: fill in position from id!
 
-  searchable :search_terms, [:title, :content, :keywords]
+  multisearchable against: %i(title_search keywords_search content_search),
+                  if: :is_faq?
 
   def to_param
     slug
   end
+
+  private
+
+  def title_search
+    Transliterator.transliterate(title&.downcase)
+  end
+
+  def content_search
+    Transliterator.transliterate(content&.downcase)
+  end
+
+  def keywords_search
+    Transliterator.transliterate(keywords&.downcase)
```

Comment:
Toto sa tak casto opakuje, ze z toho smrdi helper metoda ktora to proste spravi nad lubovolnym stringom nie?

Reply:
ok pridal som concern a tam som pridal metody